Assets/Scripts/Flashlight.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Flashlight : MonoBehaviour
+{
+    public Light flashlight;
```

Comment:
Serialized fields and tooltips perhaps? For sake of clarity on where you're getting these values from.